<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignJustify|Qt::AlignTop</set>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="serverUuidLabel">
       <property name="text">
        <string>Uuid serwera</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="serverUuidEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="serverAddressLabel">
       <property name="text">
        <string>Adres serwera</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="serverAddressEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="portEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="userNameLabel">
       <property name="text">
        <string>Użytkownik</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="userNameEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Hasło</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="passwordEdit"/>
     </item>
     <item row="6" column="1">
      <widget class="QRadioButton" name="ipv4Button">
       <property name="text">
        <string>IPv4</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QRadioButton" name="ipv6Button">
       <property name="text">
        <string>IPv6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>serverUuidEdit</tabstop>
  <tabstop>serverAddressEdit</tabstop>
  <tabstop>portEdit</tabstop>
  <tabstop>userNameEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
